dist/cytoscape.cjs.js

```diff
@@ -884,7 +884,7 @@ var uuid = function uuid(a, b
   b += a * 51 & 52 // if "a" is not 9 or 14 or 19 or 24
   ? //  return a random number or 4
   (a ^ 15 // if "a" is not 15
-  ? // genetate a random number from 0 to 15
```

Comment:
We shouldn't be including changes to the dist files in a PR

Reply:
Now removed from PR.